src/Validator/AdherentBannedValidator.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace AppBundle\Validator;
+
+use AppBundle\Entity\Adherent;
+use Doctrine\Common\Persistence\ObjectManager;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+use Symfony\Component\Validator\Exception\UnexpectedTypeException;
+use AppBundle\Entity\AdherentBanned as AdherentBanned;
+
+class AdherentBannedValidator extends ConstraintValidator
+{
+    private $objectManager;
+
+    public function __construct(ObjectManager $objectManager)
+    {
+        $this->objectManager = $objectManager;
+    }
+
+    public function validate($value, Constraint $constraint)
+    {
```

Comment:
You should also check the constraint `AdherentBanned` instance and throw a `UnexpectedTypeException`
Example :
```php
        if (!$constraint instanceof AdherentBanned) {
            throw new UnexpectedTypeException($constraint, AdherentBanned::class);
        }
```